packages/Multiselect/__tests__/__snapshots__/Multiselect.test.tsx.snap

```diff
@@ -216,13 +216,13 @@ exports[`Multiselect component should be defined and renders correctly when is o
               >
                 <div
                   class="form-field form-field--checkbox"
-                  data-cy="undefined-checkbox-cy-1"
                   id="uid-W1siQmxhemUgMiIsIm11bHRpIG1haW4gbGFiZWwiXSxbIlN1YiBsYWJlbCAxIiwic3ViIGxhYmVsIDIiLCJzdWIgbGFiZWwgMyJdXQ=="
                   role="button"
                 >
                   <input
                     checked=""
                     class="form-checkbox"
+                    data-cy="undefined-checkbox-cy-1"
```

Comment:
Is there a reason for this to be undefined?

Reply:
fixed